7062 Agrego envío de dimension a analytics para registrar la coleccion padre del item

santit96 opened the pull request:
Saco el handle de la coleccion padre a partir del trail de objetos padre del DOM con js:

![image](https://user-images.githubusercontent.com/26695177/202210365-c0e73db8-45de-44d2-bfe2-011489fc05ef.png)

![image](https://user-images.githubusercontent.com/26695177/202210526-59211dbb-2e36-4180-aab3-e5c248c09174.png)

Habría que probar que el handle se envíe bien a analytics, para eso hay que configurar en el build properties la propiedad xmlui.google.analytics.key con el UA correspondiente y luego en el navegador entrar un item y ver si se hace un POST a analytics con todas las dimensiones bien seteadas

santit96 committed:
7062 Agrego envío de dimension a analytics para registrar la coleccion padre del item
	Saco el handle de la coleccion padre a partir del trail de objetos padre del DOM con js

arieljlira commented on dspace/modules/xmlui/src/main/webapp/themes/Sedici2/lib/xsl/core/page-structure.xsl:
```diff
@@ -645,9 +645,11 @@ placeholders for header images -->
 						var itemType = document.getElementsByName("DC.type")[0].content;
 						var itemSubtype = document.getElementsByName("DC.type")[1].content;
 						var itemDateIssued = (document.getElementsByName("DCTERMS.issued")[0])? document.getElementsByName("DCTERMS.issued")[0].content : document.getElementsByName("DCTERMS.created")[0].content.substring(0,10);
+						var itemParentCollection = document.getElementById("ds-trail").querySelectorAll('.ds-trail-link:last-child')[0].firstElementChild.getAttribute('href').split('/handle/').pop();
```
Quizás en lugar de usar se puede hacer desde XSL usando datos del DRI:
```
<metadata qualifier="containerType" element="focus">type:collection</metadata>
<metadata qualifier="container" element="focus">hdl:10915/13</metadata>
```

santit96 replied:
Joya, ya lo cambié para que lo saque del DRI y no del DOM

santit96 committed:
7062 Ahora se obtiene el handle de la coleccion padre a partir del dri
	Para no estar atados al DOM que podría cambiar, ahora obtengo el handle desde el DRI, en el objeto container dentro de la seccion de pageMeta

santit96 commented:
Para el que pruebe esto hay que hacer squash de los dos commits y no merge

GarrafaGalactica commented:
Hicimos la pureba de manera local y vimos como mandaba el handle de la coleccion padre.
